hydra-plutus/scripts/vInitial.plutus

```diff
@@ -1,5 +1,5 @@
e0024603e0464a66a66666600a6ae84008894ccd5cd19b874800000854cd4cccd55cf8009281091811198149aba100235744004030426a00244a66a66666601400444a666ae68cdc3a40000042a66a6666aae7c004940988c09ccc0b8d5d08011aba200201d213500122533533333300f002225333573466e1d20000021533533335573e0024a05646464605c0046ae8800cd5d080101110a99a98088009098168008a8158a8150a8151281492814928149281490a99a9999aab9f0022502a23232302d002357440086ae8400c084854cd4cccccc0440048940b0940ac940ac8c0b0004940ac84c0b0cc0a800c004540a8540a4540a0540945409494090940909409094090854cd4cccd55cf801128129191918140011aba20043574200603842a66a601600242604e24660020080042a04a2a0482a0462a0402a0404a03e4a03e4a03e4a03e42a66a60086ae8800c84c084cd40900080045407c4c9407cc00cd5d100100a919999980100091280e9280e1280e1280e1180e8009111111999999aba40062323300735573a0026aae78004dd5003918029bab00723004375800e460066eb401c8c008dd700380ba80c181011299a800880f110999802803801180200091911180199802801000981011299a800880f11099a80f9a80111198128011981311299a800881211099a8129a8011119815801199a80091980724000002446601e00400246601c002900018020008009802000911802199a80280a8010009119b80002001301c225335001101a22133501b3500222330210023335001230090012233350090190020012300a0013004001132222323233021225335001100322133502000230040013002004300533020225335001101e221325335350032322300200b302622533500110262213500222533533010002007102913006003133502000300110013004001002301f225335001101d22133501e35002223233025003300100830252253350011355012003221350022253353300f002008112223300200a004130060033004001301b225335001101922133501a350022233020002300d0013004001301a225335001101822133501935002223301f002300c00130040013019225335001101722133501835002223301e00235500a00130040013018225335001101622133501735002220013004001301722533500110172232135003223003001301b2253350011300600322135002322253335001230043300d4800000488c014cc0380080048c010cc0340052000233500113005002101d22333573466e2000800405405884888c00c0104484888c004010c04c88d4004894cd400854cd40044054884c01d241034930370022153350031300749103493038002213500422350042253353301437666a006444466e9520003357406a0084466e952000335740602a00466ae80cc0588cd40048cdd2a400066ae80c05c004dd880d11119ba548008cd5d01ba80033357406ea0008cd5d01ba800137620380026ec4060cd5d019980aba93301575246ea000400ccd5d02999a801099ba548000dd880b1099ba548010cd5d00009bb1017213374a900119aba037520026ec405ccd5d01980a3a9001376202c0022a66a66026008004260186603c00e00a20382038464c66ae70004008480048cd40048cdd2a400066ae80dd48009bb100523374a900119aba037520026ec4014894cd400484cdd2a400066ae80c00c004dd8802899ba548008dd8802111911ba6300200130102253350011376400a4426a0044466ae80cdd8180400118038009803001a4c446a004446a00644a666ae68cdc78020010999ab9a3370e00600202001e201e44666ae68cdc7801000805805110919800801801109118010018891000980391299a800880291099a8030011802000980311299a8008803111909a801911801800980511299a800898030019109a801112999ab9a3370e900000089803001880689100109109119800802001891001091000910919800801801119180080091801119801001000a451c187ae86376f0acb38df9da73271bef525aed468dcd1771f0e98cee880001"
```

Comment:
Why did the golden files change? The changes in this PR seem to be purely on the surface (naming of functions) and any change of plutus core would is surprising to me.

Reply:
Agree this is surprising

Reply:
Agree this is surprising

Reply:
So this means that we should require the `hydra-plutus` tests to pass with these files unchanged.

Reply:
It's surprising but I don't think it's worthy of significant investigation.

Reply:
These ones are the ones that changed.

https://github.com/cardano-scaling/hydra/actions/runs/11122376981/job/30903509764?pr=1669#step:5:1589

Reply:
plutus teams says this is not unexpected, so resolving